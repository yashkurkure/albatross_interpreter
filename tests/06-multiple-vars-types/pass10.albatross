var mpJ3q6YMfLP6I8ka string := "ZKVYF";
var FeW int := 21056;
var sm7X int := FeW<=39292<>10745<FeW<=44032>33724>FeW+63574+64771|3558%FeW>10659+3961|19894<FeW+56479;
var HjpbgPA71GAIISN string := "9vCzKgPxpQP";
var CXQid8un string := "PzCJXVU";
if (32106&30963>=21648|976>sm7X<59638<>FeW||42040<>5804&&sm7X%FeW%2624&&FeW>=23356*FeW&sm7X) {while (!27139||56556+65481<>sm7X-57627*10052<>40645) {if (4524||FeW^FeW>14329+!sm7X<=36959&&4401) {return !sm7X+58046%sm7X<>60532>64720||!!15790+64072;}}}
if (FeW&23831-11937<>52192<61556>40126&sm7X&&2695<=!8246-27574>60771<6641==65020^12893-sm7X) {if (sm7X>sm7X&&FeW>FeW-28388^FeW<=FeW||FeW^!FeW+FeW<19291*FeW==54701<>29796-3235) {FeW:=19259<>41596*46130%FeW>=!39649<>sm7X>FeW|39889|sm7X>18459*20373%FeW<FeW+sm7X;}}
repeat (sm7X<=14716+22775<sm7X<4473%sm7X<46043+FeW&sm7X|28758<=sm7X||FeW|58247>=FeW+sm7X|14989) {FeW:=FeW<FeW<FeW<sm7X&29698>sm7X<>34618|FeW*22558^43742-FeW>FeW&!FeW<=!sm7X;}
mpJ3q6YMfLP6I8ka:="C";
CXQid8un:="jmI8MtHL5";
