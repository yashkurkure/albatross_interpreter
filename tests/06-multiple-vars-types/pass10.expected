Variable declared "mpJ3q6YMfLP6I8ka" type string
Variable declared "FeW" type int
Variable declared "sm7X" type int
Variable read "FeW" type int
Variable read "FeW" type int
Variable read "FeW" type int
Variable read "FeW" type int
Variable read "FeW" type int
Variable declared "HjpbgPA71GAIISN" type string
Variable declared "CXQid8un" type string
Variable read "sm7X" type int
Variable read "FeW" type int
Variable read "sm7X" type int
Variable read "FeW" type int
Variable read "FeW" type int
Variable read "FeW" type int
Variable read "sm7X" type int
Variable read "sm7X" type int
Variable read "FeW" type int
Variable read "FeW" type int
Variable read "sm7X" type int
Variable read "sm7X" type int
Variable read "sm7X" type int
Variable read "FeW" type int
Variable read "sm7X" type int
Variable read "sm7X" type int
Variable read "sm7X" type int
Variable read "sm7X" type int
Variable read "FeW" type int
Variable read "FeW" type int
Variable read "FeW" type int
Variable read "FeW" type int
Variable read "FeW" type int
Variable read "FeW" type int
Variable read "FeW" type int
Variable read "FeW" type int
Variable read "FeW" type int
Variable read "sm7X" type int
Variable read "FeW" type int
Variable read "sm7X" type int
Variable read "FeW" type int
Variable read "FeW" type int
Variable read "sm7X" type int
Variable written "FeW" type int
Variable read "sm7X" type int
Variable read "sm7X" type int
Variable read "sm7X" type int
Variable read "FeW" type int
Variable read "sm7X" type int
Variable read "sm7X" type int
Variable read "FeW" type int
Variable read "FeW" type int
Variable read "sm7X" type int
Variable read "FeW" type int
Variable read "FeW" type int
Variable read "FeW" type int
Variable read "sm7X" type int
Variable read "sm7X" type int
Variable read "FeW" type int
Variable read "FeW" type int
Variable read "FeW" type int
Variable read "FeW" type int
Variable read "sm7X" type int
Variable written "FeW" type int
Variable written "mpJ3q6YMfLP6I8ka" type string
Variable written "CXQid8un" type string
